static/app/views/settings/organization/permissionAlert.tsx

```diff
@@ -3,28 +3,33 @@ import {ReactNode} from 'react';
 import Access from 'sentry/components/acl/access';
 import Alert from 'sentry/components/alert';
 import {t} from 'sentry/locale';
+import {Organization, Scope} from 'sentry/types';
 
-type Props = React.ComponentPropsWithoutRef<typeof Alert> &
-  Pick<React.ComponentProps<typeof Access>, 'access'> & {
-    message?: ReactNode;
-  };
+type Props = React.ComponentPropsWithoutRef<typeof Alert> & {
+  access?: Scope[];
+  message?: ReactNode;
+  organization?: Organization;
+};
 
-const PermissionAlert = ({
+function PermissionAlert({
```

Comment:
I needed to update this file for one of the tests and I took the opportunity to convert it from `class` to `function`. Please let me know if that is fine